app/routes/_index.tsx

```diff
@@ -26,21 +78,202 @@ export const loader: LoaderFunction = async () => {
+				<h1 className="text-6xl font-bold text-left">
+					From individual actions to collective impact
+				</h1>
+				<h2 className="text-lg font-medium text-left py-6">
+					We enable journalists to effect real change by bringing critical
+					stories to light. Your contributions directly support this mission,
+					sustaining journalism and bolstering investigative reporting that
+					matters.
+				</h2>
+			</header>
+
+			<section className="flex flex-col lg:flex-row w-full gap-3 lg:gap-3 max-w-[1372px]">
+				<div className="flex flex-auto items-center gap-4 lg:w-[33%] rounded-3xl bg-vd-blue-200 p-4">
+					<img src={"/blue_flower.svg"} alt="blue flower drawing" />
+					<div className="flex flex-col gap-2">
+						<p className="text-base font-medium">Total Supporters</p>
+						<p className="text-3xl md:text-3xl font-bold">104</p>
+					</div>
+				</div>
+				<div className="flex flex-auto items-center gap-4 lg:w-[33%] rounded-3xl bg-vd-blue-200 p-4">
+					<img src={"/blue_elephant.svg"} alt="blue elephant drawing" />
+					<div className="flex flex-col gap-2">
+						<p className="text-base font-medium">Total Support Received</p>
+						<p className="text-3xl md:text-3xl font-bold">
+							3.6K <span className="text-lg">USD</span>
+						</p>
+					</div>
+				</div>
+				<div className="flex flex-auto items-center gap-4 lg:w-[33%] rounded-3xl bg-vd-blue-200 p-4">
+					<img src={"/blue_candle.svg"} alt="blue candle drawing" />
+					<div className="flex flex-col gap-2">
+						<p className="text-base font-medium"># of Reports Fully Funded</p>
+						<p className="text-3xl md:text-3xl font-bold">12</p>
+					</div>
+				</div>
+			</section>
+
+			<article className="w-full max-w-[1372px]">
+				<h2 className="text-3xl md:text-4xl font-semibold pt-6">Reports</h2>
+				<div className="flex flex-col md:flex-row md:justify-between md:items-end pb-8">
+					<p className="text-base pb-4 md:pb-0 ">
+						Find and fund reports that resonate with you.
+					</p>
+					<div className="flex flex-col md:flex-row gap-3">
+						<Input type="search" placeholder="Search Reports" />
+						<Select>
+							<SelectTrigger>
+								<SelectValue placeholder="Sort by" />
+							</SelectTrigger>
+							<SelectContent>
+								<SelectItem value="amount-needed">Amount Needed</SelectItem>
+								<SelectItem value="newest-oldest">Newest to Oldest</SelectItem>
+								<SelectItem value="oldest-newest">Oldest to Newest</SelectItem>
+								<SelectItem value="most-contributors">
+									Most Contributors
+								</SelectItem>
+							</SelectContent>
+						</Select>
+					</div>
+				</div>
+
+				<div className="flex flex-col md:flex-row gap-10 pb-16">
+					<section>
+						<div className="border border-b-vd-blue-400 py-4">
+							<h2 className="text-base font-medium pb-4">Categories</h2>
+							{Array.from(
+								new Set(reports.map((report: Report) => report.category)),
```

Comment:
If the reports array is going to get big there could be some optimizations here.
It might not be needed with caching but a thought.

const uniqueCategories = useMemo(() => {
  return [...new Set(reports.map((report) => report.category))];
}, [reports]);

Reply:
✅